src/NServiceBus.Core/Hosting/Helpers/AssemblyScanner.cs

```diff
@@ -377,6 +377,10 @@ static bool ReferencesNServiceBus(Assembly assembly, List<AssemblyName> processe
                 {
                     continue;
                 }
+                catch (FileLoadException)
```

Comment:
I've seen the same issue as well. So :+1: to pull it in, can we cover it with a unit test?